docs/planning/timeline-controls.mdx

```diff
@@ -0,0 +1,34 @@
+import timelineIconTray from './assets/timeline-icon-tray.png';
+
+# Timeline Controls
+
+Here we describe the timeline rendering and interaction options that are not part of the Aerie UI View Above the timeline is a tray of icon buttons that can be used to control how data is rendered and interacted with on the timeline.
+
+<figure>
+  <img alt="Aerie UI - Timeline Icon Tray" src={timelineIconTray} />
+</figure>
+
+The following controls are available:
+
+- `Auto Scroll to Offscreen Selections` - Automatically scrolls the timline to the selected activity directive or span. This can be helpful when selecting directives or spans from their associated tables.
+- `Decimation` - Perform on the fly min-max decimation of line layer points for improved timeline rendering performance. Learn more about decimation below.
+- `Cursor Value Interpolation` - Show interpolated value when hovering over resources. Learn more about interpolation below.
+- `Limit Tooltip to Line` - When enabled, the timeline tooltip will only appear when hovering near a resource line segment. When disabled, the tooltip will appear when hovering over any part of a resource layer.
+- `Timeline Tooltip Visibility` - Toggle timeline tooltip visibility.
+- `Timeline Modes` - Toggle between `Pan & Zoom` and `Interaction` modes. Learn more about timeline modes below.
```

Comment:
Minor, nonblocking: consider adding a link to that lower section?

Reply:
What's the best way to add a link to another section here?

Reply:
Standard Markdown style, ie

```markdown
[Learn more about timeline modes below](#timeline-modes).
```

Reply:
D'oh